msal/managed_identity.py

```diff
@@ -563,7 +598,12 @@ def _obtain_token_on_service_fabric(
     logger.debug("Obtaining token via managed identity on Azure Service Fabric")
     resp = http_client.get(
         endpoint,
-        params={"api-version": "2019-07-01-preview", "resource": resource},
+        params={k: v for k, v in {
+            "api-version": "2019-07-01-preview",
```

Comment:
```suggestion
            "api-version": "2019-07-01-preview",
```
We just need a final confirmation on this one @rayluo. App Service is setting a new api-version, we want a sign off from SF

Reply:
What kind of sign-off do we need? A confirmation written in email? Or as a PR? (This is my genuine question, because I want to be more effective on this kind of things.)

In this case, I chatted with SF via Teams channel. I can add you into that group chat upon request.

Reply:
I do it [this way](https://identitydivision.visualstudio.com/Engineering/_workitems/edit/3184155), so we have tracking items closed and partner's agreement captured

Reply:
> I do it [this way](https://identitydivision.visualstudio.com/Engineering/_workitems/edit/3184155), so we have tracking items closed and partner's agreement captured

That work item is now in Done status, but where can we find the partner's agreement captured? The api version topic seems not mentioned, neither in that work item, nor in [the spec that work item referenced](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/5137/files).

Reply:
@gladjohn , now that we have that [final confirmation](https://identitydivision.visualstudio.com/Engineering/_workitems/edit/3184155), can we resolve this and re-review this PR?